css/src/components/code-block.scss

```diff
@@ -0,0 +1,68 @@
+$code-font-size: $font-size-8 !default;
+$code-block-spacing: 1em !default;
+$code-header-min-height: 2em !default;
+$code-header-color: $text !default;
+$code-block-header-font-size: 0.8em !default;
+$code-header-body-line-height: 1.3571 !default;
+
+@mixin code-block {
+	pre {
+		box-sizing: border-box;
+		max-width: 100%;
+		margin: 0;
+		padding: $code-block-spacing;
+		border: 1px solid $border;
+		background-color: $code-block;
+		font-size: $code-font-size;
+		line-height: 1.3571;
+		white-space: pre;
+		word-spacing: normal;
+		overflow: auto;
+		font-smooth: auto;
```

Comment:
I noticed that `font-smooth` and `-webkit-overflow-scrolling` doesn't seem to be working (would need to be `webkit-font-smoothing`/`-moz-osx-font-smoothing`). It's the same in Docs-UI. Should we remove them as they're non-standard? 

![image](https://user-images.githubusercontent.com/64044661/115443030-9547b300-a1c7-11eb-9b70-b45b46ea3631.png)

https://developer.mozilla.org/en-US/docs/Web/CSS/font-smooth
https://developer.mozilla.org/en-US/docs/Web/CSS/-webkit-overflow-scrolling

Reply:
Re: overflow scrolling: My understanding is that its just an additive on iOS safari, so it makes sense that it would never apply on a desktop machine.

Reply:
Re: font-smoothing - great catch! I would have expected auto-prefixer to handle this one, but it seems it does not. I've added the vendor prefixes.